src/components/Button/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import React, {FC, useCallback} from 'react';
+import {TouchableHighlight, Text} from 'react-native';
+import styles from './styles';
+import {ButtonColor} from './Button.types';
+
+type Props = {
+  buttonColor: ButtonColor;
+  text: string;
+  onPressHandler: Function;
+};
+
+const Button: FC<Props> = ({buttonColor, text, onPressHandler}) => {
+  const onPress = useCallback(() => {
+    onPressHandler();
+  }, [onPressHandler]);
```

Comment:
if we use `onPress` in this component - there is no need to use `useCallback` here, also `onPressHandler` is came from props. So this is unnecessary